scripts/kmodtool

```diff
@@ -182,9 +182,17 @@ BuildRequires:	  kernel-devel-uname-r = ${kernel_uname_r}
 %{?KmodsRequires:Requires: %{KmodsRequires}-uname-r = ${kernel_uname_r}}
 %{?KmodsRequires:BuildRequires: %{KmodsRequires}-uname-r = ${kernel_uname_r}}
 %post          -n kmod-${kmodname}-${kernel_uname_r}
-${prefix}${depmod_path} -aeF /boot/System.map-${kernel_uname_r} ${kernel_uname_r} > /dev/null || :
+if [[ -f "/boot/System.map-${kernel_uname_r}" ]]; then
+	${prefix}${depmod_path} -aeF /boot/System.map-${kernel_uname_r} ${kernel_uname_r} > /dev/null || :
+else
```

Comment:
```suggestion
elif [[ -f /lib/modules/${kernel_uname_r}/System.map ]]; then
```
It seems like it would be a good idea to also check for the file in this case.  Then perhaps have a generic `else` block which just runs `depmod -a ${kernel_uname_r}` as a default fallback.

Reply:
Fixed